app/admin/books.rb

```diff
@@ -0,0 +1,61 @@
+ActiveAdmin.register Book do
+  permit_params :title, :description, :price_cents, :dimensions, :year_of_publication, :materials, :quantity,
+                category_ids: [], author_ids: []
+
+  decorate_with BookDecorator
+
+  filter :authors, collection: AuthorDecorator.decorate_collection(Author.all)
```

Comment:
and here

Reply:
In order to print author full_name i used decorator.